src/deliver-mobile/generate-distribute-mobile-app/publish-apple-app-store.md

```diff
@@ -45,7 +45,7 @@ If you don’t have one, learn [how you can create a certificate](https://develo
 * You  need a provisioning profile according to the developer program you enrolled in:   
 — `App Store` provisioning profile for the Apple Developer Program, or  
 — `In-House` provisioning profile for the Apple Developer Enterprise Program  
-A provisioning profile allows your application to be launched on Apple devices and use app services. If you don't have one, learn how you can [create a provisioning profile](https://help.apple.com/xcode/mac/current/#/devaafd622d2). For more information on the different provisioning profiles, check the Medium blog post "[Cruising through the Complexities of Signing Native Mobile Apps](<https://medium.com/outsystems-engineering/cruising-through-the-complexities-of-signing-native-mobile-apps-cc123eb2814b>)" by OutSystems.
+A provisioning profile allows your application to be launched on Apple devices and use app services. If you don't have one, learn how you can [create a provisioning profile](https://help.apple.com/xcode/mac/current/#/devaafd622d2). If you already have a provisioning profile **make sure it has a valid expiring date**. If you are using an In-House provisioning profile don't forget to **rebuild and redistribute your app when the provisioning profile is renewed**. You can find more details about provisioning profiles and In-House distribution [here](https://support.apple.com/en-au/guide/deployment-reference-ios/apda0e3426d7/1/web/1.0). For more information on the different provisioning profiles, check the Medium blog post "[Cruising through the Complexities of Signing Native Mobile Apps](<https://medium.com/outsystems-engineering/cruising-through-the-complexities-of-signing-native-mobile-apps-cc123eb2814b>)" by OutSystems.
```

Comment:
Maybe this line can be broke down in more bullets, because this information is really useful and important. Or give other kind of visibility

Reply:
I reduced the amount of new information

Reply:
So, numbering the sentences in line 48:
1. > If you already have a provisioning profile **make sure it has a valid expiring date**. 

2. > If you are using an In-House provisioning profile don't forget to **rebuild and redistribute your app when the provisioning profile is renewed**. 

3. > You can find more details about provisioning profiles and In-House distribution [here](https://support.apple.com/en-au/guide/deployment-reference-ios/apda0e3426d7/1/web/1.0). 

Sentence 2 is valid for any provisioning profile not only In House, so I am not sure if we should have it there. I believe that 1 and 3 are enough, being the third one the most critical one, because it adds the link to the official Apple documentation regarding In-House distribution